### PR TITLE
Update docstrings of MaskRCNN

### PR DESCRIPTION
**Pull request by hmk114**
The original docstring of MaskRCNN class says the returned mask is a `UInt8Tensor` in inference mode, which is not true. This can be verified by the following code:

```python
import torch
from torchvision.models.detection import maskrcnn_resnet50_fpn_v2

model = maskrcnn_resnet50_fpn_v2()
model.eval()

with torch.no_grad():
    output = model([torch.randn(3, 100, 100)])

print(output[0]["masks"].dtype) # torch.float32
```

### CODE REVIEW

**Review by NicolasHug — APPROVED** [review]
Thanks @hmk114
